File: day3/Python/can.drive.py
age = int(input("How old are you? "))
if age >= 18:
    print("You can drive.") 
elif age >= 15:
    print("You can drive with a learner's permit.")
    print("You must be accompanied by a licensed driver.")
    print("You cannot drive yet.")
elif age < 0:
    print("Invalid age entered. Please enter a valid age.")
else:
    print("You cannot drive yet, you little shit!")

File: day3/JS/candrive.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("How old are you? ", (age) => {
    console.log("Before parsing, type of age:", typeof age);
  age = parseInt(age);
    console.log("After parsing, type of age:", typeof age);
  if (age >= 18) {
    console.log("You can drive!");
  } else if (age >= 15) {
    console.log("You can get a learner's permit.");
  } else {
    console.log("Too young to drive.");
  }
  readline.close();
});